pkgs/racket-doc/syntax/scribblings/modcode.scrbl

```diff
@@ -52,10 +57,14 @@ The @racket[rkt-try-ss?] argument defaults to @racket[#t].  If it is not
 @racket[#f], then if @racket[path] ends in @filepath{.rkt}, then the
 corresponding file ending in @filepath{.ss} will be tried as well.
 
-The @racket[choose-proc] argument is a procedure that takes three
-paths: a source path, a @filepath{.zo} file path, and an extension path
-(for a non-@tt{_loader} extension). Some of the paths may not
-exist. The result should be either @racket['src], @racket['zo],
+If the @racket[allow-zo-so] argument is @racket[#f], then the code is
+always loaded from source, never from a @filepath{.zo } or @filepath{.so}
```

Comment:
Extra space after `.zo`. The "extension" suffix is not always `.so` (it's `.dylib` on Mac OS and `.dll` on Windows), so the word "extension" is probably more right there.

Reply:
As in ".zo file or extension"?

Reply:
Yes. It looks like "dynamic extension" is the defined term, so `@tech{dynamic extension}` instead of just `extension`.